src/Tickets/Blocks/Tickets/app/editor/container/template.js

```diff
@@ -46,42 +48,66 @@ const TicketsContainer = ({
 		return null;
 	}
 
+	const renderPromptInfo = () => applyFilters(
```

Comment:
This is probably the correct application of the filter.
Add the missing doc-block.
It seems wasteful to build the `<PromptInfo />` component (which does not belong here since it's really a Series Passes prompt info) and then nullify it in a filter.
I think firing an action here would make more sense.
I think an even better approach would be to have a `null` prompt info by default in the mapped properties to let filtering functions add the prompt info that might be required.

In the case at hand, Flexible Tickets could filter the `tec.tickets.blocks.Tickets.TicketsContainer.mappedProps` filter, add the prompt when required.

Note - I've updated the filtering code to allow for React and JSX in the filters file, to smooth this kind of operation out: pull the latest version of the feature branch to get it.

Reply:
Thanks @lucatume , I applied your suggestion in filterTicketsContainerMappedProps and `null` is displayed by default in the template